# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../madwifi/madwifi-driver_nodepmod.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- madwifi/Makefile.orig	2005-06-24 10:00:38.000000000 -0700
+++ madwifi/Makefile	2005-06-24 10:01:12.000000000 -0700
@@ -56,13 +56,13 @@
 	for i in ${DIRS}; do \
 		(cd $$i; make install) || exit 1; \
 	done
-	@if [ -z $(DESTDIR) ]; then \
-	    /sbin/depmod -ae ; \
-	elif [ -f $(SYSTEMMAP) ]; then \
-	    /sbin/depmod -ae -b $(DESTDIR) -F $(SYSTEMMAP) $(KERNELRELEASE) ; \
-	else \
-	    echo "Don't forget to run \"depmod -ae\" on the target system."; \
-	fi
+	#@if [ -z $(DESTDIR) ]; then \
+	#    /sbin/depmod -ae ; \
+	#elif [ -f $(SYSTEMMAP) ]; then \
+	#    /sbin/depmod -ae -b $(DESTDIR) -F $(SYSTEMMAP) $(KERNELRELEASE) ; \
+	#else \
+	#    echo "Don't forget to run \"depmod -ae\" on the target system."; \
+	#fi
 
 FILES=	ath_hal COPYRIGHT ath include Makefile Makefile.inc \
 	README release.h share net80211 ath_rate
